## Problema 3
A petición de Ingeniería de Procesos para poder cubrir la inusual demanda que están teniendo, el Equipo de Gestión de la Energía 
propone un análisis de mercado para evaluar si el recambio tecnológico del motor de una cinta transportadora es rentable en la relación
costo-beneficio (si la demanda luego baja, no recuperamos más la inversión, murmuraron hacia el final de la reunión).
El motor de 30 kW y con etiqueta IE1, en el período enero-mayo, funcionaba durante 10 hs y el horómetro marcaba 4 hs 
a plena carga. En el trimestre siguiente (de junio a septiembre), el horómetro comenzó a registrar el doble de tiempo hs a plena carga.  

El líder del Equipo le solicita:
    
a) estimar la energía útil entregada por el motor diaria en el período de enero-mayo.
b) estimar la energía secundaria consumida por el motor diaria en el período de enero-mayo (emplear la diapositiva 64 considerando que rend(f = 0.5)) = rend(f = 1) = 0.84 para este motor).
c) ídem a para el período junio-septiembre.
d) ídem b para el período junio-septiembre.
e) elabore un análisis (de no más de un párrafo) con los resultados anteriores.
f) "por esas casualidades" se le ocurre sugerir un certificado de calibración del horómetro. Se lleva a laboratorio y el informe luego indica que no estaba calibrado. 
¿Cómo le parece que impacta esta situación ante el recambio planteado?


In [11]:
pot100 = 30   # Potencia nominal en kW
pot50 = pot100 / 2  # Potencia al 50% de carga. Como se trata de una aplicación a par constante, la curva de carga es lineal con la velocidad.
eta_E1 = 0.84    # Eficiencia promedio E1 según enunciado
dias_1er = 3 * 31 + 28 +30  # Suma de los días del 1er trimestre - Enero a Mayo
dias_1er
horas_100 = 4 * dias_1er   # 4 horas al 100% de carga
horas_100
energia_100 = horas_100 * pot100  # Energia con carga al 100% en el primer trimestre
energia_100
print("Energia al 100% de carga")
print(energia_100)
horas_50 = 6 * dias_1er  # Se asume que el resto de las horas trabaja al 50% de carga.
energia_50 = horas_50 * pot50
print("Energia al 50% de carga")
print(energia_50)
energia_util = (energia_100 + energia_50) * eta_E1
print("Energia util primer trimestre:")
print(energia_util)
print("Energia Diaria util primer trimestre:")
print(energia_util/dias_1er)

Energia al 100% de carga
18120
Energia al 50% de carga
13590.0
Energia util primer trimestre:
26636.399999999998
Energia Diaria util primer trimestre:
176.39999999999998


### Solución Item a)

La energía útil entregada por el motor diaria en el período de enero-mayo es de **176.4 kWh**

### Solución Item b)

Estimar la energía secundaria consumida por el motor diaria en el período de enero-mayo (emplear la diapositiva 64 considerando que rend(f = 0.5)) = rend(f = 1) = 0.84 para este motor).

Se asume que el rendimiento promedio es del 84%, y el factor de potencia del motor es 0,95.

Luego, la energía secundaria diaria para el primer trimestre es de **221.05 kVA**


In [13]:
PF = 0.95  # Factor de potencia supuesto
energia_sec = (energia_util/(dias_1er * PF * eta_E1))   # Energía secundaria en kVA
print("Energía secundaria diaria primer trimestre")
print(energia_sec)

Energía secundaria diaria primer trimestre
221.05263157894737
